packages/mgt-components/src/components/mgt-person-card/sections/mgt-person-card-contact/mgt-person-card-contact.ts

```diff
@@ -93,7 +93,7 @@ export class MgtPersonCardContact extends BasePersonCardSection {
       title: this.strings.chatTitle
     },
     businessPhone: {
-      icon: getSvg(SvgIcon.CellPhone, '#605e5c'),
+      icon: getSvg(SvgIcon.Phone, '#929292'),
```

Comment:
This should be using the same color as the other icons,
```suggestion
      icon: getSvg(SvgIcon.Phone, '#605e5c'),
```